packages/fuels/src/cli/utils/fuelsVersionCache.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs';
+import path from 'path';
+
+export const FUELS_VERSION_CACHE_FILE = path.join(__dirname, 'FUELS_VERSION');
```

Comment:
I believe we should add this to the `.gitignore` file

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3350/commits/d595dbf9712ee76b88d09a64c729d303d5a52d02